src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -323,12 +323,12 @@ void GenerateErrorOrWarning ()
 				if (!string.IsNullOrEmpty (file))
 					Log.LogWarning ("", "XA0102", "", file, line, column, 0, 0, text.Replace ("Warning:", ""));
 				else 
-					Log.LogWarning (text.Replace ("Warning:", ""));
+					Log.LogCodedWarning ("XA0102", text.Replace ("Warning:", ""));
```

Comment:
Would you care to write some [message documentation](https://github.com/xamarin/xamarin-android/tree/master/Documentation/guides/messages) for XA0102 and XA0103? :-D

Reply:
@jonpryor done and done